src/types/events/base-events.ts

```diff
@@ -882,11 +885,14 @@ export interface UserChangeEvent {
       status_text: string;
       status_text_canonical: string;
       status_emoji: string;
+      status_emoji_display_info: [];
```

Comment:
Can you change this empty array type to be the correct one? Same for other events. https://github.com/slackapi/node-slack-sdk/blob/%40slack/web-api%406.7.1/packages/web-api/src/response/UsersInfoResponse.ts#L87

Reply:
Thanks for catching that @seratch ! I think I have fixed that now.